front-end/src/components/general/BaseButton.jsx

```diff
@@ -33,7 +33,7 @@ const BaseButton = (props) => {
                 onClick={props.click}
                 className={getButtonStyle()}
                 disabled={props.isDisabled}
-                aria-label={props.label}
+                aria-label={props.text || props.label}
```

Comment:
good idea but maybe had it default to text not label since label is a property designed specifically for the label?